libs/state/src/lib/rx-state.service.ts

```diff
@@ -94,8 +95,69 @@ export class RxState<T extends object> implements OnDestroy, Subscribable<T> {
    *
    * @return T
    */
-  get(): T {
-    return this.accumulator.state;
+
+  get(): T;
+
+  /**
+   * @description
+   * Read from the state in imperative manner by providing keys as parameters.
+   * Returns the part of state object.
+   *
+   * @example
+   * // Access a single property
+   *
+   * const bar = state.get('bar');
+   *
+   * // Access a nested property
+   *
+   * const foo = state.get('bar', 'foo');
+   *
+   * @return T | Partial<T>
+   */
+
+  get<K1 extends keyof T>(k1?: K1): T | Partial<T>;
+
+  get<K1 extends keyof T, K2 extends keyof T[K1]>(
+    k1?: K1,
+    k2?: K2
+  ): T | Partial<T>;
```

Comment:
isnt it this?
```suggestion
  ): T | T[K1][K2];
```

Reply:
https://github.com/BioPhoton/rx-angular/pull/161/commits/715df6a0f63a7e724a6ebe5bda19a8b3f6c5c1cf